### PR TITLE
add in a runtime example

### PR DESCRIPTION
**Pull request by andymckay**
Coverage for:

```
chrome.runtime.onInstalled
chrome.runtime.getManifest
chrome.runtime.reload
```

From #36

### CODE REVIEW

File: runtime-examples/manifest.json [hunk 1]
```diff
@@ -0,0 +1,17 @@
+{
+  "browser_action": {
+    "default_icon": "run.png"
+  },
+  "description": "Some runtimes example",
+  "homepage_url": "https://github.com/mdn/webextensions-examples/tree/master/tabs-tabs-tabs",
```
Comment (wbamberg) [inline runtime-examples/manifest.json]: This should be updated.

File: runtime-examples/manifest.json [hunk 2]
```diff
@@ -0,0 +1,17 @@
+{
+  "browser_action": {
+    "default_icon": "run.png"
+  },
+  "description": "Some runtimes example",
```
Comment (wbamberg) [inline runtime-examples/manifest.json]: Maybe something a bit more complete, like "Demonstration of some runtime APIs."

File: runtime-examples/manifest.json [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+  "homepage_url": "https://github.com/mdn/webextensions-examples/tree/master/tabs-tabs-tabs",
+  "manifest_version": 2,
+  "name": "Runtime examples",
+  "version": "1.0",
+  "permissions": ["notifications"],
+  "background": {
+    "scripts": ["background.js"]
+  },
+  "icons": {
+    "128": "run.png"
```
Comment (wbamberg) [inline runtime-examples/manifest.json]: great icon :)

File: runtime-examples/background.js [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+    type: 'basic'
+  });
+}
+
+function onClick() {
+  browser.runtime.reload();
+}
+
+browser.browserAction.setBadgeText({
+  text: `${loadTime.getMinutes()}:${loadTime.getSeconds()}`
```
Comment (wbamberg) [inline runtime-examples/background.js]: This gets truncated at 4 characters, so it's confusing (It shows something like "47.2"). I don't think you really need this loadTime/badge text stuff, do you? I'd be tempted to omit it.

Comment (andymckay) [reply runtime-examples/background.js]: Really? For me it shows all 5 characters. I wonder why.

We don't really, I just wanted to show the effect of reload in some way. I could add it to the notification message instead where it won't be truncated.

File: runtime-examples/manifest.json (hunk 1 above)
Comment (andymckay) [reply runtime-examples/manifest.json]: Oops, thanks.

File: runtime-examples/background.js (hunk 4 above)
Comment (wbamberg) [reply runtime-examples/background.js]: That is weird.

>  I just wanted to show the effect of reload in some way.

I think seeing the notification again works for that, doesn't it?

Comment (andymckay) [reply runtime-examples/background.js]: It shows that the install notification has been called, but not that the background page (and other stuff) has been reloaded as well.

**Pull request merged**